src/crypto/index.js

```diff
@@ -1018,26 +1030,26 @@ Crypto.prototype.backupGroupSession = function(
         throw new Error("Key backups are not enabled");
     }
 
-    const data = this._backupPayloadForSession(
-        senderKey, forwardingCurve25519KeyChain,
-        sessionId, sessionKey, keysClaimed,
-        exportFormat,
-    );
-    return this._baseApis.sendKeyBackup(roomId, sessionId, this.backupInfo.version, data);
+    await this._cryptoStore.markSessionsNeedingBackup([{
+        senderKey: senderKey,
+        sessionId: sessionId,
+    }]);
+
+    this._maybeSendKeyBackup();
```

Comment:
probably worth `await`ing on this too? Actually possibly not: when we call it from megolm we only want to wait for it to be saved to indexeddb, not actually backed up. worth a comment though.

Reply:
Actually, I think it does make sense to await here, and then remove the awaits from megolm.